Backend/routes/auth.js

```
@@ -0,0 +1,26 @@
+const express= require('express');
+const bcrypt= require('bcryptjs');
+const jwt= require('jsonwebtoken');
+const User= require('../models/User');
+
+const router= express.Router();                                        //to register new user
+router.post('/register', async(req, res) => {
+    const { username, password}=req.body;
+    const hashedPassword=await bcrypt.hash(password, 10);
+    const User= new User({username, password: hashedPassword });
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)